### DISCUSSION TITLE
How to save the plotted images in svg format in pycaret 3.0 version?

### DISCUSSION THREAD

**Discussion by moon195** [pycaret/pycaret#3709] [General]
SVG is a vector format which makes it easier to keep images clear when publishing articles. However, the default image format saved in PyCaret is PNG. How can I modify it to directly save images in SVG format?

**Comment by celestinoxp** [comment]
Please provide more details. Do you mean interpret_model? I don't think there is an option to choose the type of formatting.
There are 2 things you can do:
1 - Use an external tool to convert the images. Example: https://www.img2go.com/convert-to-image
2 - Open an Issue with a request to implement the function to choose the format.

**Comment by moon195** [comment]
When I was redoing the plots for the regression model, I noticed that the default saving format is PNG. How can I change the default format? An example  is shown below
“plot_model(lightgbm, plot = 'residuals',save=True)
'Residuals.png'”
